schema.sql

```diff
@@ -14,48 +16,90 @@ price real NOT NULL,
+  INSERT INTO products
+    (url, name, rating, reviews, price, isPrime)
+  VALUES
+    ('Carrying+Case.jpg', 'AmazonBasics Carrying Case for Nintendo Switch - Black', 3.9, 350, 13.99, true);
+  INSERT INTO products
+    (url, name, rating, reviews, price, isPrime)
+  VALUES
+    ('My+Little+Pony.jpg', 'My Little Pony Apple Jack Costume Zip-Up Hoodie', 4.5, 24, 18.99, true);
+  INSERT INTO products
+    (url, name, rating, reviews, price, isPrime)
+  VALUES
+    ('Food+Storage.jpg', 'Sistema KLIP IT Collection Food Storage Containers, Assorted Shapes and Colors, 20-Piece Set', 3.8, 357, 29.99, true);
+  INSERT INTO products
+    (url, name, rating, reviews, price, isPrime)
+  VALUES
+    ('Dyson.jpg', 'Dyson Articulating Hard Floor Tool', 4.1, 1593, 39.63, true);
+  INSERT INTO products
+    (url, name, rating, reviews, price, isPrime)
+  VALUES
+    ('SMART+BUY.jpg', 'SMART BUY 400 G3 PD DM I5-6500T', null, null, 649.99, true);
+  INSERT INTO products
+    (url, name, rating, reviews, price, isPrime)
+  VALUES
+    ('ACER.jpg', 'Acer G226HQL 21.5-Inch Screen LED Monitor', 4, 6062, 71.70, true);
+  INSERT INTO products
+    (url, name, rating, reviews, price, isPrime)
+  VALUES
+    ('Echo+Spot.jpg', 'Echo Spot - Black', 4.6, 4355, 129.99, true);
+  INSERT INTO products
+    (url, name, rating, reviews, price, isPrime)
+  VALUES
+    ('BOSE.jpg', 'Bose QuietComfort 35 (Series II) Wireless Headphones, Nosie Cancelling - Black', 4.4, 1118, 349.00, true);
+  INSERT INTO products
+    (url, name, rating, reviews, price, isPrime)
+  VALUES
+    ('Polk+Audio.jpg', 'Polk Audio CS10 Center Channel Speaker (Single, Black)', 4.3, 900, 99.00, false);
+  INSERT INTO products
+    (url, name, rating, reviews, price, isPrime)
+  VALUES
+    ('Harry+Potter.jpg', 'Harry Potter: Complete 8-Film Collection', 4.5, 2753, 50.87, true);
+  INSERT INTO products
+    (url, name, rating, reviews, price, isPrime)
+  VALUES
+    ('Yamaha+Bass.jpg', 'Yamaha BB Series 4-String Bass - Tabacco Sunburst', 5, 10, 766.71, false);
+  INSERT INTO products
+    (url, name, rating, reviews, price, isPrime)
+  VALUES
+    ('Electric+Guitar.jpg', 'ESP LTD EC-356 Electric Guitar, BlackEl', 4, 16, 399.00, true);
+  INSERT INTO products
+    (url, name, rating, reviews, price, isPrime)
+  VALUES
+    ('Basketball.jpg', 'Wilson Evolution Indoor Game Basketball', 4.7, 1483, 42.63, false);
+  INSERT INTO products
+    (url, name, rating, reviews, price, isPrime)
+  VALUES
+    ('Air+Force+1.jpg', 'Nike Men Air Force 1 2007', 1, 1, 120.51, false);
+  INSERT INTO products
+    (url, name, rating, reviews, price, isPrime)
+  VALUES
+    ('Football.jpg', 'Wilson NFL Junior Touchdown Football', 5, 36, 9.99, true);
+  INSERT INTO products
+    (url, name, rating, reviews, price, isPrime)
+  VALUES
+    ('Imploding+Kittens.jpg', 'Imploding Kittens: This is the First Expansion of Exploding Kittens', 4.6, 942, 14.99, true);
 
```

Comment:
You can do `INSERT INTO _table_name_  (_column_names_) VALUES (_set_of_values_), (_another_set_of_values_);` if you want as well instead of having `INSERT INTO _table_name_  (_column_names_)` repeated every time. Doesn't matter since this is already done, just here to let you know.

Reply:
Thanks for the heads up!